app.py

```
@@ -1,4 +1,75 @@
 from flask import Flask, render_template, request, redirect, url_for
 import sqlite3
 import subprocess  # To run the facial recognition script
```

Ruff (0.8.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)